common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -79,7 +79,7 @@ def do_my_expensive_update(course_key):
        do the actual work.
 
     """
-    course_published = django.dispatch.Signal(providing_args=["course_key"])
+    course_published = django.dispatch.Signal(providing_args=["course_key", "item_keys"])
```

Comment:
Could you please put a comment clarifying exactly when this signal is published and when handlers should expect to get something in `item_keys` or not? Are there ever cases where we double-signal (one with items and one without)? Can the number of items in `item_keys` grow to be every item in the course (because we change some inherited field)?